### PR TITLE
modifications for CentOS-7

### PR DESCRIPTION
**Pull request by electrical**
Centos-7 uses Systemd

### CODE REVIEW

File: lib/beaker/host_prebuilt_steps.rb [hunk 1]
```diff
@@ -350,6 +350,8 @@ def enable_root_login host, opts
         #restart sshd
         if host['platform'] =~ /debian|ubuntu|cumulus/
           host.exec(Command.new("sudo su -c \"service ssh restart\""), {:pty => true})
+        elsif host['platform'] =~ /centos-7|el-7|redhat-7/
```
Comment (petems) [inline lib/beaker/host_prebuilt_steps.rb]: Fedora > 15 is SystemD too I believe :+1:

File: lib/beaker/host_prebuilt_steps.rb [hunk 2]
```diff
@@ -506,6 +508,9 @@ def set_env host, opts
         when /debian|ubuntu|cumulus/
           host.exec(Command.new("echo '\nPermitUserEnvironment yes' >> /etc/ssh/sshd_config"))
           host.exec(Command.new("service ssh restart"))
+        when /el-7|centos-7|redhat-7|oracle-7|scientific-7|eos-7/
```
Comment (petems) [inline lib/beaker/host_prebuilt_steps.rb]: Same here

**Comment by electrical** [comment]
I would like to take the fedora modifications into an other PR, want to move the sshd restarting into a separate function since its used in multiple places.

**Comment by anodelman** [comment]
test this please

**Comment by anodelman** [comment]
Please associate your commit with a beaker ticket number (BKR-xxx).

**Comment by anodelman** [comment]
ok to test

**Comment by anodelman** [comment]
Review comments still current.

**Comment by electrical** [comment]
Will process tomorrow ( Tuesday )

**Comment by anodelman** [comment]
retest this please

**Pull request merged**